src/instance.ts

```diff
@@ -83,7 +86,9 @@ export type SetInstanceMetadataCallback = ResourceCallback<
   GaxOperation,
   IOperation
 >;
-export interface GetInstanceConfig extends GetConfig {}
+export interface GetInstanceConfig
+  extends GetConfig,
+    GetInstanceMetadataOptions {}
```

Comment:
nits: can we put `GetInstanceMetadataOptions {}` into previous line? like: 

```typescript
export interface GetInstanceConfig
  extends GetConfig, GetInstanceMetadataOptions {}
```

Reply:
I tried, but unfortunately linter is complaining about it.
`npm run fix` breaks it back in two lines.